packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.service.js

```diff
@@ -127,55 +132,76 @@ export default class DedicatedServerInterfacesService {
 
   getTasks(serverName) {
     return this.$http
-      .get(`/dedicated/server/${serverName}/task?function=${INTERFACE_TASK}`)
-      .then(
-        (response) => response.data,
-        () => [],
-      )
-      .then((taskIds) => ({
-        promise: this.waitAllTasks(
-          serverName,
-          taskIds.map((taskId) => ({ taskId })),
-        ),
-      }));
+      .get(`/dedicated/server/${serverName}/task`, {
+        headers: {
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
+          'X-Pagination-Size': 5000,
+          'X-Pagination-Filter': `function:in=${INTERFACE_TASK},${INTERFACE_GROUP_TASK},${INTERFACE_UNGROUP_TASK}`,
```

Comment:
It is better to use native route filters instead of Iceberg one, if possible to do so